<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>View</class>
 <widget class="QMainWindow" name="View">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1175</width>
    <height>701</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1175</width>
    <height>701</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1175</width>
    <height>701</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>3D Viewer v2</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="openFile">
    <property name="geometry">
     <rect>
      <x>2</x>
      <y>2</y>
      <width>109</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
   border: 2px solid gray;
  border-radius: 8px;
}</string>
    </property>
    <property name="text">
     <string>Open</string>
    </property>
    <property name="default">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="scp::GLWidget" name="openGLWidget" native="true">
    <property name="geometry">
     <rect>
      <x>2</x>
      <y>68</y>
      <width>800</width>
      <height>600</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QComboBox" name="projectionType">
    <property name="geometry">
     <rect>
      <x>226</x>
      <y>2</y>
      <width>149</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QComboBox{
   border: 2px solid gray;
  border-radius: 8px;
}</string>
    </property>
   </widget>
   <widget class="QPushButton" name="takeScreenshot">
    <property name="geometry">
     <rect>
      <x>377</x>
      <y>2</y>
      <width>132</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
   border: 2px solid gray;
  border-radius: 8px;
}</string>
    </property>
    <property name="text">
     <string>Screenshot</string>
    </property>
    <property name="default">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="getGIF">
    <property name="geometry">
     <rect>
      <x>511</x>
      <y>2</y>
      <width>87</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
   border: 2px solid gray;
  border-radius: 8px;
}</string>
    </property>
    <property name="text">
     <string>GIF</string>
    </property>
    <property name="default">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="renderFile">
    <property name="geometry">
     <rect>
      <x>113</x>
      <y>2</y>
      <width>111</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
   border: 2px solid gray;
  border-radius: 8px;
}</string>
    </property>
    <property name="text">
     <string>Render</string>
    </property>
    <property name="default">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QFrame" name="frame_2">
    <property name="geometry">
     <rect>
      <x>816</x>
      <y>499</y>
      <width>350</width>
      <height>171</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <layout class="QGridLayout" name="gridLayout_4">
     <property name="rightMargin">
      <number>5</number>
     </property>
     <item row="1" column="3">
      <widget class="QLabel" name="labelVerticesColor">
       <property name="text">
        <string>Color:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="labelVerticesDisplay">
       <property name="text">
        <string>Type:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1" colspan="2">
      <widget class="QLabel" name="labelVerticesSize">
       <property name="text">
        <string>Size:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QRadioButton" name="squareVerticeType">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Square</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0" rowspan="2">
      <widget class="QRadioButton" name="noneVerticeType">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>None</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0" colspan="4">
      <widget class="QLabel" name="labelVerticesStyle">
       <property name="text">
        <string>Vertices style</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QRadioButton" name="circleVerticeType">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Circle</string>
       </property>
      </widget>
     </item>
     <item row="2" column="3" rowspan="4">
      <widget class="QPushButton" name="verticeColor">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>72</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="default">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="2" column="1" rowspan="4" colspan="2">
      <widget class="QSlider" name="verticeSize">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>50</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>816</x>
      <y>360</y>
      <width>350</width>
      <height>140</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <layout class="QGridLayout" name="gridLayout_3">
     <property name="rightMargin">
      <number>5</number>
     </property>
     <item row="0" column="0" colspan="4">
      <widget class="QLabel" name="labelLineStyle">
       <property name="text">
        <string>Line style</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="1" column="3">
      <widget class="QLabel" name="labelLineColor">
       <property name="text">
        <string>Color:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0" rowspan="2">
      <widget class="QRadioButton" name="solidLineType">
       <property name="text">
        <string>Solid</string>
       </property>
      </widget>
     </item>
     <item row="2" column="3" rowspan="3">
      <widget class="QPushButton" name="lineColor">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>72</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="default">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="labelLineType">
       <property name="text">
        <string>Type:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1" colspan="2">
      <widget class="QLabel" name="labelLineThickness">
       <property name="text">
        <string>Thickness:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QRadioButton" name="dashedLineType">
       <property name="text">
        <string>Dashed</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1" rowspan="3" colspan="2">
      <widget class="QSlider" name="lineThickness">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>151</width>
         <height>0</height>
        </size>
       </property>
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>100</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QFrame" name="frame_3">
    <property name="geometry">
     <rect>
      <x>816</x>
      <y>308</y>
      <width>350</width>
      <height>53</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="leftMargin">
      <number>11</number>
     </property>
     <property name="topMargin">
      <number>5</number>
     </property>
     <property name="rightMargin">
      <number>5</number>
     </property>
     <property name="bottomMargin">
      <number>5</number>
     </property>
     <item>
      <widget class="QLabel" name="labelBackgroundColor">
       <property name="text">
        <string>Background color</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="bgColor">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>72</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="default">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QFrame" name="frame_5">
    <property name="geometry">
     <rect>
      <x>816</x>
      <y>68</y>
      <width>350</width>
      <height>71</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <layout class="QGridLayout" name="gridLayout_2">
     <property name="leftMargin">
      <number>5</number>
     </property>
     <property name="topMargin">
      <number>5</number>
     </property>
     <property name="rightMargin">
      <number>5</number>
     </property>
     <property name="bottomMargin">
      <number>5</number>
     </property>
     <item row="0" column="0">
      <widget class="QLabel" name="numVerticesText">
       <property name="text">
        <string>Number of vertices</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="numVertices">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="numEdgesText">
       <property name="text">
        <string>Number of edges</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLabel" name="numEdges">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QFrame" name="frame_4">
    <property name="geometry">
     <rect>
      <x>816</x>
      <y>138</y>
      <width>350</width>
      <height>171</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <property name="leftMargin">
      <number>5</number>
     </property>
     <property name="topMargin">
      <number>5</number>
     </property>
     <property name="rightMargin">
      <number>5</number>
     </property>
     <property name="bottomMargin">
      <number>5</number>
     </property>
     <item row="2" column="0">
      <widget class="QLabel" name="move_text">
       <property name="text">
        <string>Move (x, y, z)</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0" colspan="2">
      <widget class="QLabel" name="rotate_text">
       <property name="text">
        <string>Rotate (x, y, z)</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QDoubleSpinBox" name="scale_on_k">
       <property name="minimum">
        <double>-1000.000000000000000</double>
       </property>
       <property name="maximum">
        <double>1000.000000000000000</double>
       </property>
       <property name="value">
        <double>1.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="3" column="3" colspan="2">
      <widget class="QPushButton" name="resetAffine">
       <property name="styleSheet">
        <string notr="true">QPushButton {
   border: 2px solid gray;
  border-radius: 6px;
}</string>
       </property>
       <property name="text">
        <string>Reset</string>
       </property>
       <property name="default">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="0" column="0" colspan="2">
      <widget class="QLabel" name="scale_text">
       <property name="text">
        <string>Scale (k)</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QDoubleSpinBox" name="rotate_x">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="baseSize">
        <size>
         <width>71</width>
         <height>0</height>
        </size>
       </property>
       <property name="layoutDirection">
        <enum>Qt::LeftToRight</enum>
       </property>
       <property name="minimum">
        <double>-1000.000000000000000</double>
       </property>
       <property name="maximum">
        <double>1000.990000000000009</double>
       </property>
       <property name="value">
        <double>0.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QDoubleSpinBox" name="move_on_x">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="baseSize">
        <size>
         <width>71</width>
         <height>0</height>
        </size>
       </property>
       <property name="layoutDirection">
        <enum>Qt::LeftToRight</enum>
       </property>
       <property name="minimum">
        <double>-1000.000000000000000</double>
       </property>
       <property name="maximum">
        <double>1000.000000000000000</double>
       </property>
       <property name="value">
        <double>0.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="2" column="3">
      <widget class="QDoubleSpinBox" name="move_on_y">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="baseSize">
        <size>
         <width>71</width>
         <height>0</height>
        </size>
       </property>
       <property name="layoutDirection">
        <enum>Qt::LeftToRight</enum>
       </property>
       <property name="minimum">
        <double>-1000.000000000000000</double>
       </property>
       <property name="maximum">
        <double>1000.000000000000000</double>
       </property>
       <property name="value">
        <double>0.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="1" column="4">
      <widget class="QDoubleSpinBox" name="rotate_z">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="baseSize">
        <size>
         <width>71</width>
         <height>0</height>
        </size>
       </property>
       <property name="layoutDirection">
        <enum>Qt::LeftToRight</enum>
       </property>
       <property name="minimum">
        <double>-1000.000000000000000</double>
       </property>
       <property name="maximum">
        <double>1000.990000000000009</double>
       </property>
       <property name="value">
        <double>0.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="1" column="3">
      <widget class="QDoubleSpinBox" name="rotate_y">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="baseSize">
        <size>
         <width>71</width>
         <height>0</height>
        </size>
       </property>
       <property name="layoutDirection">
        <enum>Qt::LeftToRight</enum>
       </property>
       <property name="minimum">
        <double>-1000.000000000000000</double>
       </property>
       <property name="maximum">
        <double>1000.990000000000009</double>
       </property>
       <property name="value">
        <double>0.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="2" column="4">
      <widget class="QDoubleSpinBox" name="move_on_z">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="baseSize">
        <size>
         <width>71</width>
         <height>0</height>
        </size>
       </property>
       <property name="layoutDirection">
        <enum>Qt::LeftToRight</enum>
       </property>
       <property name="minimum">
        <double>-1000.000000000000000</double>
       </property>
       <property name="maximum">
        <double>1000.000000000000000</double>
       </property>
       <property name="singleStep">
        <double>1.000000000000000</double>
       </property>
       <property name="value">
        <double>0.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="3" column="0" colspan="3">
      <widget class="QPushButton" name="applyAffine">
       <property name="styleSheet">
        <string notr="true">QPushButton {
   border: 2px solid gray;
  border-radius: 6px;
}</string>
       </property>
       <property name="text">
        <string>Apply</string>
       </property>
       <property name="default">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="labelProgName">
    <property name="geometry">
     <rect>
      <x>816</x>
      <y>0</y>
      <width>350</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
      <italic>true</italic>
      <bold>true</bold>
      <underline>false</underline>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: #19c257</string>
    </property>
    <property name="text">
     <string>3D Viewer v2</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="labelAuthors">
    <property name="geometry">
     <rect>
      <x>816</x>
      <y>40</y>
      <width>350</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <italic>true</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: #19c257</string>
    </property>
    <property name="text">
     <string>by elidacon, ternesim</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="undo_button">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>2</y>
      <width>98</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
   border: 2px solid gray;
  border-radius: 8px;
}</string>
    </property>
    <property name="text">
     <string>Undo</string>
    </property>
    <property name="icon">
     <iconset theme="edit-undo">
      <normaloff>.</normaloff>.</iconset>
    </property>
    <property name="default">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="redo_button">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>2</y>
      <width>98</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
   border: 2px solid gray;
  border-radius: 8px;
}</string>
    </property>
    <property name="text">
     <string>Redo</string>
    </property>
    <property name="icon">
     <iconset theme="edit-redo">
      <normaloff>.</normaloff>.</iconset>
    </property>
    <property name="default">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QComboBox" name="filePath">
    <property name="geometry">
     <rect>
      <x>2</x>
      <y>34</y>
      <width>798</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QComboBox{
   border: 2px solid gray;
  border-radius: 8px;
}</string>
    </property>
    <property name="currentText">
     <string/>
    </property>
   </widget>
   <zorder>frame_5</zorder>
   <zorder>openFile</zorder>
   <zorder>openGLWidget</zorder>
   <zorder>projectionType</zorder>
   <zorder>takeScreenshot</zorder>
   <zorder>getGIF</zorder>
   <zorder>filePath</zorder>
   <zorder>renderFile</zorder>
   <zorder>frame_2</zorder>
   <zorder>frame</zorder>
   <zorder>frame_3</zorder>
   <zorder>frame_4</zorder>
   <zorder>labelProgName</zorder>
   <zorder>labelAuthors</zorder>
   <zorder>undo_button</zorder>
   <zorder>redo_button</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1175</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>1</height>
    </size>
   </property>
   <property name="baseSize">
    <size>
     <width>0</width>
     <height>1</height>
    </size>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>scp::GLWidget</class>
   <extends>QWidget</extends>
   <header>./view/scp_glwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>filePath</tabstop>
  <tabstop>openFile</tabstop>
  <tabstop>renderFile</tabstop>
  <tabstop>projectionType</tabstop>
  <tabstop>takeScreenshot</tabstop>
  <tabstop>getGIF</tabstop>
  <tabstop>scale_on_k</tabstop>
  <tabstop>rotate_x</tabstop>
  <tabstop>rotate_y</tabstop>
  <tabstop>rotate_z</tabstop>
  <tabstop>move_on_x</tabstop>
  <tabstop>move_on_y</tabstop>
  <tabstop>move_on_z</tabstop>
  <tabstop>applyAffine</tabstop>
  <tabstop>resetAffine</tabstop>
  <tabstop>bgColor</tabstop>
  <tabstop>solidLineType</tabstop>
  <tabstop>dashedLineType</tabstop>
  <tabstop>lineThickness</tabstop>
  <tabstop>lineColor</tabstop>
  <tabstop>noneVerticeType</tabstop>
  <tabstop>circleVerticeType</tabstop>
  <tabstop>squareVerticeType</tabstop>
  <tabstop>verticeSize</tabstop>
  <tabstop>verticeColor</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
